### PR TITLE
Pangolin 3764 localized text field

### PR DESCRIPTION
**Pull request by Sarah4VT**
#### Summary

Update the localized text field component to support an isCondensed property that is passed down to the localized text Input to display the condensed layout.

## Description

[Figma Design](https://www.figma.com/file/PaImaWorWz1RbV85gqw4MH/Filter-%26-Table-controls?type=design&node-id=1305-135823&mode=design&t=E8vYa0ewveJolju1-4)
image

Results
image

### CODE REVIEW

**Comment by Sarah4VT** [comment]
Do not merge this until [this pr](https://github.com/commercetools/ui-kit/pull/2791) is merged first.

File: packages/components/fields/localized-text-field/src/localized-text-field.tsx [hunk 1]
```diff
@@ -145,6 +145,10 @@ export type TLocalizedTextFieldProps = {
    * Focus the input on initial render
    */
   isAutofocussed?: boolean;
+  /**
+   * Whether the input and its options are rendered with condensed paddings
```
Comment (CarlosCortizasCT) [inline packages/components/fields/localized-text-field/src/localized-text-field.tsx:149]: There are no options here so perhaps we can adjust the description.
Suggestion:
```
Whether the text inputs for each localization should render with condensed paddings.
```

_(please remember to regenerate the READMEs)_

Comment (Sarah4VT) [reply packages/components/fields/localized-text-field/src/localized-text-field.tsx]: I updated this based on your suggestion and ran the script to regenerate readme's. Evidently we were skipping that step on several of these tickets. So there is more included than just this in this pr.

**Review by ddouglasz — APPROVED** [review]
Looks good to me,
Thank you

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks Sarah 🙇

**Review by chloe0592 — APPROVED** [review]
Thank you! 👍

**Comment by CarlosCortizasCT** [comment]
@FilPob could we please have your review? 🙏

**Review by FilPob — APPROVED** [review]

**Pull request merged**